app/views/contests/_index.html.erb

```diff
@@ -27,8 +26,7 @@
     <td><%= contest.name %></td>
     <td><%= contest.start_time.strftime("%b %d, %H:%M") unless contest.start_time.nil? %></td>
     <td><%= contest.end_time.strftime("%b %d, %H:%M") unless contest.end_time.nil? %></td>
-    <td><%= contest.duration %></td>
-    <td><%= contest.owner_id %></td>
+    <td><%= contest.duration %> hours</td>
```

Comment:
Self reminder to correctly pluralise this

Reply:
I think this is fine actually, "1.0 hours" sounds more correct than "1.0 hour".
https://english.stackexchange.com/questions/44489/is-1-0-singular-or-plural